docs/cloud/managing-airbyte-cloud/manage-schema-changes.md

```diff
@@ -1,12 +1,27 @@
 # Manage schema changes
 
-Once every 24 hours, Airbyte checks for changes in your source schema and allows you to review the changes and fix breaking changes. This process helps ensure accurate and efficient data syncs, minimizing errors and saving you time and effort in managing your data pipelines.
+You can specify for each connection how Airbyte should handle any change of schema in the source. This process helps ensure accurate and efficient data syncs, minimizing errors and saving you time and effort in managing your data pipelines.
 
-:::note 
+Airbyte checks for any changes in your source schema before syncing, at most once every 24 hours.
 
-Schema changes are flagged in your connection but are not propagated to your destination.
-    
-:::
+Based on your configured settings for "Detect and propagate schema changes", Airbyte can automatically sync those changes or ignore them: 
+* **Propagate all changes** automatically propagates stream changes (additions or deletions) or column changes (additions or deletions) detected in the source
+* **Propagate column changes only** automatically propagates column changes detected in the source
+* **Ignore** any schema change, in which case the schema you’ve set up will not change even if the source schema changes until you approve the changes manually
+* **Pause connection** disables the connection from syncing further once a change is detected
+
+When a new column is detected and propagated, values for that column will be filled in for the updated rows. If you are missing values for rows not updated, a backfill can be done by completing a full refresh.
+
+When a column is deleted, the values for that column will stop updating for the updated rows and be filled with Null values.
```

Comment:
I think that we will actually now delete these columns immediately, based on the related convo in Normalization.

Reply:
@alex-gron Was thinking about this more today after our chat - I agree that's the behavior we decided on. Does that mean the proposed changes will come with Destinations V2 - which I believe that's coming at the end of Q3? On the one hand, we could publish these with the foresight those changes will be coming, or re-publish with the updates when V2 is officially released. I'm leaning towards the latter but let me know if you think it's better to just update now!

Reply:
Sorry for the late reply! I'm good with that option :)